docs/code/cgi/main.c

```diff
@@ -0,0 +1,81 @@
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <uv.h>
+
+uv_loop_t *loop;
```

Comment:
So, I assume this is taken directly from the book right? It should be `uv_loop_t* loop`, but are we going to go through and lint the whole thing by hand?